docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -16,7 +16,6 @@ Using `next` ensures that it will always use the latest v6 alpha release, but yo
 ## Run codemods
 
 The `preset-safe` codemod will automatically adjust the bulk of your code to account for breaking changes in v6. You can run `v6.0.0/data-grid/preset-safe` targeting only Data Grid or `v6.0.0/preset-safe` to target Date and Time pickers as well.
-It should be only applied **once per folder.**
```

Comment:
I think this doesn't make sense as almost all our codemods (at least preset-safe ones) are supposed to be `idempotent` so this line may confuse the user in case of some errors on the first run. It should be OK to run it any number of time as user wishes.

CC @LukasTy @alexfauquette

Reply:
Good point. 👍 
It indeed makes little sense having this sentence, especially including **bold**. 🙈